modules/sildisco/public/sp/saml2-logout.php

```diff
@@ -58,8 +58,9 @@
 
 \SimpleSAML\Module\saml\Message::validateMessage($idpMetadata, $spMetadata, $message);
 
+$httpUtils = new \SimpleSAML\Utils\HTTP();
```

Comment:
I really like to import with use statements to keep dependencies more visible at the top of the file.

Reply:
Good suggestion. I'll make a note to do that across the board.